client/server/debug.go

```diff
@@ -310,6 +314,20 @@ func (s *Server) addCommonConfigFields(configContent *strings.Builder) {
 	configContent.WriteString(fmt.Sprintf("BlockLANAccess: %v\n", s.config.BlockLANAccess))
 }
 
+func (s *Server) addGoroutines(req *proto.DebugBundleRequest, anonymizer *anonymize.Anonymizer, archive *zip.Writer) error {
+	var buff []byte
+	myBuff := bytes.NewBuffer(buff)
+	err := pprof.Lookup("goroutine").WriteTo(myBuff, 2)
```

Comment:
Instead, we should keep the pprof format (0), which makes debugging the output much easier with the `go tool pprof`.

```suggestion
	err := pprof.Lookup("goroutine").WriteTo(myBuff, 0)
```